nextflow.config

```diff
@@ -94,16 +113,22 @@ params {
 // Load base.config by default for all pipelines
 includeConfig 'conf/base.config'
 
-// Load modules.config for DSL2 module specific options
-includeConfig 'conf/modules.config'
-
 // Load nf-core custom profiles from different Institutions
 try {
     includeConfig "${params.custom_config_base}/nfcore_custom.config"
 } catch (Exception e) {
     System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")
 }
 
+// Load nf-core/airrflow custom profiles from different institutions.
```

Comment:
Remove this maybe?

Reply:
it is like this commented in the template so I would prefer leaving it as the template to avoid conflicts